src/shared/components/rightbar/RightBar.tsx

```diff
@@ -49,7 +49,10 @@ export default class RightBar extends React.Component<{}, {}> {
                     <h3>Cancer Studies</h3>
                     {
                         (this.studyStore.data.isComplete) && (
-                            <BarGraph data={ this.studyStore.data.result }/>
+                            <a href="/data_sets.jsp">
```

Comment:
this needs to be "data_sets.jsp", no leading slash.  The reason is that some instances of portal are inside subfolders (e.g. cbioportal.org/beta/index.do).  Between you and me this is not a good practice to host instances this way, but it's the reality.  So all links must be relative.